src/components/Navbar/Navbar.jsx

```diff
@@ -22,6 +25,14 @@ const Navbar = () => {
             : header.classList.remove("scroll-header");
     };
 
+    useEffect(() => {
+        if (window.location.pathname !== "/") {
+            setNavItemColor("#2d3748");
+        } else {
+            setNavItemColor("#cbd5e1");
+        }
+    }, [location]);
+
     useEffect(() => {
         window.addEventListener("scroll", scrollHeader);
```

Comment:
you can do simply window.scrollTo(0,0). avoid giving events to the document. And for this functionality you can make custom hook and re-use it

Reply:
Thanks, I have used hook to change navbar color, but I can't use `window.scrollTo(0,0)` as it doesn't serve the same purpose I want.